lib/trento_web/controllers/v1/database_json.ex

```diff
@@ -1,48 +1,45 @@
-defmodule TrentoWeb.V1.DatabaseView do
-  use TrentoWeb, :view
+defmodule TrentoWeb.V1.DatabaseJSON do
+  def databases(%{databases: databases}), do: Enum.map(databases, &database(%{database: &1}))
 
-  def render("databases.json", %{databases: databases}) do
-    render_many(databases, __MODULE__, "database.json", as: :database)
-  end
-
-  def render("database.json", %{
+  def database(%{
         database:
           %{
             database_instances: database_instances
           } = database
-      }) do
-    rendered_database_instances =
-      render_many(database_instances, __MODULE__, "database_instance.json", as: :instance)
-
-    database
-    |> Map.from_struct()
-    |> Map.delete(:__meta__)
-    |> Map.delete(:deregistered_at)
-    |> Map.delete(:sap_systems)
-    |> Map.delete(:tenants)
-    |> Map.put(:database_instances, rendered_database_instances)
-    |> add_system_replication_status_to_secondary_instance
-  end
-
-  def render("database_instance.json", %{instance: %{database_id: database_id} = instance}) do
-    instance
-    |> Map.from_struct()
-    # keep backward compatibility
-    |> Map.put(:sap_system_id, database_id)
-    |> Map.delete(:__meta__)
-    |> Map.delete(:host)
-  end
-
-  def render("database_instance_with_sr_status.json", %{
+      }),
+      do:
+        database
+        |> Map.from_struct()
+        |> Map.delete(:__meta__)
+        |> Map.delete(:deregistered_at)
+        |> Map.delete(:sap_systems)
+        |> Map.delete(:tenants)
+        |> Map.put(
+          :database_instances,
+          Enum.map(database_instances, &database_instance(%{instance: &1}))
+        )
+        |> add_system_replication_status_to_secondary_instance
+
+  def database_instance(%{instance: %{database_id: database_id} = instance}),
+    do:
```

Comment:
Props for the `, do:`

Reply:
For me personally, it looks worst this way :see_no_evil: 
Not for being one liner we should go this way. I would use the `, do:` in really easy function declarations. (More if you even have comments in the code)
Anyway, just my opinion

Reply:
Well the question is, what  looks better for the eye. 

```
  def database_instance(%{instance: %{database_id: database_id} = instance}) do
    instance
    |> Map.from_struct()
    # keep backward compatibility
    |> Map.put(:sap_system_id, database_id)
    |> Map.delete(:__meta__)
    |> Map.delete(:host)
  end
```

vs

```
  def database_instance(%{instance: %{database_id: database_id} = instance}),
    do:
      instance
      |> Map.from_struct()
      # keep backward compatibility
      |> Map.put(:sap_system_id, database_id)
      |> Map.delete(:__meta__)
      |> Map.delete(:host)
```

It is almost the same with the pipe operator, just one line less without the end block. 
For me i prefer the shorter version, both works perfectly,  but i think we should just decide and have the same way in the whole code base :)

Reply:
I would say this is subjective (as elixir allow both options), but for me, one liners are for one liners, like having something like:
```
def function(), do: do_one_line_things
# or
def some_long_function(), do:
  do_one_line_things
``` 

Otherwise, it is not obvious to your eye where a function start and ends.
Again, just me opinion.
And I don't think we should extend on this and have a "strict code rule"